benchmark/win.ipynb

```diff
@@ -0,0 +1,3654 @@
+       "│\u001b[36m \u001b[0m\u001b[36m        Valid Matches\u001b[0m\u001b[36m \u001b[0m│\u001b[35m \u001b[0m\u001b[35m635               \u001b[0m\u001b[35m \u001b[0m│\n",
+       "│\u001b[36m \u001b[0m\u001b[36mPlackettLuce Accuracy\u001b[0m\u001b[36m \u001b[0m│\u001b[35m \u001b[0m\u001b[35m583/52 [ 91.81%]  \u001b[0m\u001b[35m \u001b[0m│\n",
+       "│\u001b[36m \u001b[0m\u001b[36m   OpenSkill Duration\u001b[0m\u001b[36m \u001b[0m│\u001b[35m \u001b[0m\u001b[35m1.6394069194793701\u001b[0m\u001b[35m \u001b[0m│\n",
+       "│\u001b[36m \u001b[0m\u001b[36m   TrueSkill Accuracy\u001b[0m\u001b[36m \u001b[0m│\u001b[35m \u001b[0m\u001b[35m593/42 [ 93.39%]  \u001b[0m\u001b[35m \u001b[0m│\n",
+       "│\u001b[36m \u001b[0m\u001b[36m   TrueSkill Duration\u001b[0m\u001b[36m \u001b[0m│\u001b[35m \u001b[0m\u001b[35m5.139376878738403 \u001b[0m\u001b[35m \u001b[0m│\n",
+       "│\u001b[36m \u001b[0m\u001b[36m          Speedup (%)\u001b[0m\u001b[36m \u001b[0m│\u001b[35m \u001b[0m\u001b[35m 213.49           \u001b[0m\u001b[35m \u001b[0m│\n",
+       "│\u001b[36m \u001b[0m\u001b[36m    Accuracy Bump (%)\u001b[0m\u001b[36m \u001b[0m│\u001b[35m \u001b[0m\u001b[35m-1.69             \u001b[0m\u001b[35m \u001b[0m│\n",
+       "└───────────────────────┴────────────────────┘\n"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "table = Table(title=\"Benchmark Results\")\n",
+    "table.add_column(\"Information\", justify=\"right\", style=\"cyan\", no_wrap=True)\n",
+    "table.add_column(\"Value\", style=\"magenta\")\n",
+    "\n",
+    "table.add_row(\"Available Matches\", f\"{available_matches}\")\n",
+    "table.add_row(\"Valid Matches\", f\"{valid_matches}\")\n",
+    "\n",
+    "openskill_accuracy = round(\n",
+    "    (\n",
+    "        openskill_correct_predictions\n",
+    "        / (openskill_incorrect_predictions + openskill_correct_predictions)\n",
+    "    )\n",
+    "    * 100,\n",
+    "    2,\n",
+    ")\n",
+    "\n",
+    "trueskill_accuracy = round(\n",
+    "    (\n",
+    "        trueskill_correct_predictions\n",
+    "        / (trueskill_incorrect_predictions + trueskill_correct_predictions)\n",
+    "    )\n",
+    "    * 100,\n",
+    "    2,\n",
+    ")\n",
+    "\n",
+    "table.add_row(\n",
+    "    f\"{model.__class__.__name__} Accuracy\",\n",
+    "    f\"{openskill_correct_predictions}/\"\n",
+    "    f\"{openskill_incorrect_predictions} \"\n",
+    "    f\"[{openskill_accuracy: .2f}%]\",\n",
+    ")\n",
+    "table.add_row(\"OpenSkill Duration\", f\"{openskill_time}\")\n",
+    "\n",
+    "table.add_row(\n",
+    "    f\"TrueSkill Accuracy\",\n",
+    "    f\"{trueskill_correct_predictions}/\"\n",
+    "    f\"{trueskill_incorrect_predictions} \"\n",
+    "    f\"[{trueskill_accuracy: .2f}%]\",\n",
+    ")\n",
+    "table.add_row(\"TrueSkill Duration\", f\"{trueskill_time}\")\n",
+    "speedup = ((trueskill_time - openskill_time) / openskill_time) * 100\n",
```

Comment:
```suggestion
    "speedup = (trueskill_time  / openskill_time) * 100\n",
```

I would just report speedup as ratio of times as that is more standard and in line with what is reported in paper (I was initially confused why I was seeing what I thought was a ~2x speedup when a 3x speedup was reported).

Reply:
Will be fixed in the latest change.